sqlstore/src/main/resources/db/migration/V8000__container_constraints.sql

```diff
@@ -0,0 +1,14 @@
+--StartNoTest
+ALTER TABLE Run_SequencerPartitionContainer ADD FOREIGN KEY (Run_runId) REFERENCES Run (runId);
+DELETE FROM Run_SequencerPartitionContainer where containers_containerId = 0;
+ALTER TABLE Run_SequencerPartitionContainer ADD FOREIGN KEY (containers_containerId) REFERENCES SequencerPartitionContainer (containerId);
+
+ALTER TABLE SequencerPartitionContainer_Partition ADD FOREIGN KEY (container_containerId) REFERENCES SequencerPartitionContainer (containerId);
+ALTER TABLE SequencerPartitionContainer_Partition ADD FOREIGN KEY (partitions_partitionId) REFERENCES _Partition (partitionId);
+
+ALTER TABLE SequencerPartitionContainer ADD UNIQUE (identificationBarcode);
```

Comment:
This will also fail on our production db, as we have two containers with identificationBarcode `C8G67` (containers 48 and 512). Both of these containers are associated with pools, though only container 48 is associated with a run. What do you think will be the best course of action for handling this duplicate?